src/react/atlascode/pipelines/PipelineSummaryPage.tsx

```diff
@@ -484,7 +488,15 @@ const PipelineSummaryPage: React.FunctionComponent = () => {
                 </div>
             );
         },
-        [classes.loglessStepHeader, classes.stepHeader, controller.fetchLogs, stepHeader],
+        [
+            classes.loglessStepHeader,
+            classes.stepHeader,
+            classes.hiddenOverflow,
+            classes.logHeader,
+            classes.logs,
+            controller.fetchLogs,
+            stepHeader,
+        ],
```

Comment:
Looking at these changes in the PR view, I'm seriously leaning towards just silencing linter warnings instead ;(